auctions/english-auction/README.md

```diff
@@ -1,56 +1,148 @@
-        <img alt="light theme" src=".docs/english-auction_light.png">
-    </picture>
-</p>
-
-## Overview
-
-An English Auction is where a seller auctions off an asset with an initial price and a reserve price. Users will then begin bidding on the asset until the bidding period has ended or the reserve has been met. Upon completion, users will withdraw either their original deposits or their newly purchased assets depending on the outcome.
-
-The English Auction application implements this idea in a decentralized manner without the need for a 3rd party and with strong settlement assurances. The application has been designed to utilize native assets and NFTs enabling users to auction off native assets / NFTs and place bids using native assets / NFTs. 
-
-More information can be found in the [specification](./SPECIFICATION.md).
-
-## Repository Structure
-
-The project consists of a smart contract.
-
-```
-english-auction/
-├── project/
-|   └── auction-contract/
-|       ├── src/main.sw
-|       └── tests/harness.rs
-├── README.md
-└── SPECIFICATION.md
-```
-
-## Running the project
-
-### User Interface
-
-TODO: UI does not currently exist
-
-### Tests
-
-In order to run the tests make sure that you are in the root of this project i.e. `/path/to/auctions/english-auction/<you are here>`
-
-Build the contracts:
-
-```bash
-forc build
-```
-
-Run the tests:
-
-```bash
-cargo test
-```
-
-## Specification
-
-The specification contains a non-technical overview of the contract indicating the flow of information from the start to the end of the english-auction.
-
-Check [SPECIFICATION.md](./SPECIFICATION.md) for more info!
\ No newline at end of file
+<p align="center">
+    <picture>
+        <source media="(prefers-color-scheme: dark)" srcset=".docs/english-auction_dark.png">
+        <img alt="light theme" src=".docs/english-auction_light.png">
+    </picture>
+</p>
+
+## Overview
+
+An English Auction is where a seller auctions off an asset with an initial price and a reserve price. Users will then begin bidding on the asset until the bidding period has ended or the reserve has been met. Upon completion, users will withdraw either their original deposits or their newly purchased assets depending on the outcome.
+
+The English Auction application implements this idea in a decentralized manner without the need for a 3rd party and with strong settlement assurances. The application has been designed to utilize native assets and NFTs enabling users to auction off native assets / NFTs and place bids using native assets / NFTs.
+
+More information can be found in the [specification](./SPECIFICATION.md).
+
+## Repository Structure
+
+The project consists of a smart contract.
+
+```
+english-auction/
+├── docker/
+├── packages
+|   └── app/
+|   └── config/
+|   └── contracts/project/auction-contract/
+|       ├── src/main.sw
+|       └── tests/harness.rs
+|   └── scripts/
+├── README.md
+└── SPECIFICATION.md
```

Comment:
hm, I kind of like this structure in a way too but I prefer to isolate the contents more i.e. separate the UI from the contracts similar to [this](https://github.com/FuelLabs/sway-applications/tree/master/.template#project-structure). Don't need to do this now though but before `master` a decision will need to be made since all the other apps are structured around that and the tools that @supiket is working on, afaik, are based on the repo structure

Reply:
Ok makes sense.  The other structure is still similar to mine.  The app folder could be renamed to ui, the contracts folder could be renamed to fit the other structure as well.  The scripts folder is a useful tool for frontend development work, which will eventually get put in the ts sdk.  The config folder is only used to configure and build the scripts package so that should go away as well.  I like the packages folder because it separates the packages from the docker folder, which makes sense as the docker folder does not represent a package.

Reply:
yh, docker on the top level is likely fine